src/sentry/models/event.py

```diff
@@ -195,23 +150,15 @@ def ip_address(self):
 
         return None
 
-    def get_interfaces(self):
-        return CanonicalKeyView(get_interfaces(self.data))
-
-    @memoize
-    def interfaces(self):
-        return self.get_interfaces()
-
-    def get_tags(self):
```

Comment:
This method is used in a few places in sentry-plugins. You might need to keep it around as an alias or update sentry-plugins.

Reply:
Good point, done.